src/Stripe.net/Services/UsageRecordSummaries/UsageRecordSummaryService.cs

```diff
@@ -5,7 +5,8 @@ namespace Stripe
     using System.Threading.Tasks;
     using Stripe.Infrastructure;
 
-    public class UsageRecordSummaryService : BasicService<UsageRecord>
+    public class UsageRecordSummaryService : BasicService<UsageRecord>,
+        INestedListable<UsageRecordSummary, UsageRecordSummaryListOptions>
```

Comment:
Are you sure this one is correct? This service is weird because it does not take the record id as a first parameter. It likely should though and we should fix it but it makes me think this current setup would not work?